### DISCUSSION TITLE
Konfiguration von Preset-Werten je Strukturelement

### DISCUSSION THREAD

**Discussion by andre-hohmann** [kitodo/kitodo-production#5825] [Q&A]
**Beschreibung**
Im Regelsatz ist es möglich, je `<key>` einen Preset-Wert zu definieren. Siehe: [Voreingestellte Werte für Metadatenschlüssel](https://github.com/kitodo/kitodo-production/wiki/Regelsatz-3.x#voreingestellte-werte-f%C3%BCr-metadatenschl%C3%BCssel)

In einigen Fällen wäre es hilfreich, wenn auch je Strukturelement Preset-Werte definiert werden könnten. 
Es ist möglich, über `permit`, die Werte einzuschränken. Jedoch wird, wenn nur ein Wert definiert wird, dieser zwar in dem Feld angezeigt, allerdings nicht in die METS-Datei eingetragen. 
Dadurch ist die Validierung nicht korrekt, obwohl es für die anwendenden Personen korrekt aussieht. 
Zudem kann der Wert nicht mehr manuell ausgewählt werden, so dass eine Korrektur nicht mehr möglich ist. 

**Frage** 
Ist es möglich, einen Preset-Wert je Strukturelement zu bestimmen, so dass der Wert ohne manuellen Aufwand eingetragen wird?

**Beispiel** 
Konfiguration
```
<permit key="docType" maxOccurs="1"  minOccurs="1" unspecified="forbidden">
    <permit value="NewspaperYear"/>
</permit>
```
Anzeige
![WertStrukturelement02](https://github.com/kitodo/kitodo-production/assets/14835770/027bc5c6-bd03-41b4-888d-bb75e68854c0)

![WertStrukturelement01](https://github.com/kitodo/kitodo-production/assets/14835770/4128d207-9e9f-4a22-98ea-4284f09a1d8d)

**Comment by andre-hohmann** [comment]
@matthias-ronge : Ich vermute, dass Du am ehesten antworten kannst. 
Fällt Dir noch eine andere Möglichkeit ein, um auch ohne `preset` Funktion, Werte ohne manuelle Bestätigung in ein bestimmtes Feld einzutragen?

**Reply by matthias-ronge** [reply]
Gefühlt ist das für mich ein Bug. Wenn ein Wert angezeigt wird, muss er auch in die METS-Datei eingetragen werden.
